test/unit/helpers/dom/MfaVerifyForm.js

```diff
@@ -182,6 +182,14 @@ define(['./Form'], function (Form) {
 
     signoutLink: function ($sandbox) {
       return $sandbox.find('[data-se=signout-link]');
+    },
+
+    passwordToggleShowContainer: function() {
+      return this.$('.password-toggle span.button-show');
```

Comment:
nit: Just `.password-toggle .button-show` should do. We don't want to add too much dependency to the DOM.